public/locales/es/translation.json

```diff
@@ -0,0 +1,193 @@
+{
+  "header": {
+      "title": "Sunburst Smorgasbord",
+      "subtitle": "Anarquía relacional, tan radiante como el sol."
+  },
+  "button": {
+      "download_image": "Descargar como imagen",
+      "download_json": "Exportar",
+      "edit": "Editar",
+      "import_json": "Importar",
+      "reset": "Reiniciar"
+  },
+  "edit": {
+      "add": "Añadir",
+      "add_as_child": "Añadir como hijo",
+      "close": "Cerrar",
+      "customize": "Personaliza tu Smorgasbord",
+      "new_flavour": "Nuevo sabor",
```

Comment:
This is a very direct translation, I'm not sure if it makes sense. It doesn't tell me much. Though "flavour" also doesn't tell me that much and I don't have a better suggestion.

@duizendnegen could you tell me some synonyms of "flavour" that would work for you?

Reply:
Category, preference - it comes from that this wheel is based on a coffee flavour wheel

Reply:
Categoría then. We are not used to coffee flavour wheels :)

Reply:
I've just noticed that we can use "Añadir como Subcategoría" instead of "hijo", to keep it neutral.